owncloudApp/src/androidTest/java/com/owncloud/android/settings/more/SettingsFragmentMoreSectionTest.kt

```diff
@@ -0,0 +1,380 @@
+            prefAboutApp = fragment.findPreference(PREFERENCE_ABOUT_APP)!!
+        }
+
+        prefSecurityCategory.isVisible = false
+    }
+
+    @Test
+    fun moreView() {
+        launchTest()
+
+        onView(withText(R.string.prefs_category_more)).check(matches(isDisplayed()))
+        assertEquals(PREFERENCE_MORE_CATEGORY, prefMoreCategory.key)
+        assertEquals(context.getString(R.string.prefs_category_more), prefMoreCategory.title)
+        assertEquals(null, prefMoreCategory.summary)
+        assertTrue(prefMoreCategory.isVisible)
+
+        onView(withText(R.string.prefs_help)).check(matches(isDisplayed()))
+        assertEquals(PREFERENCE_HELP, prefHelp?.key)
+        assertEquals(context.getString(R.string.prefs_help), prefHelp?.title)
+        assertEquals(null, prefHelp?.summary)
+        assertTrue(prefHelp?.isVisible == true)
+        assertTrue(prefHelp?.isEnabled == true)
+
+        onView(withText(R.string.prefs_sync_calendar_contacts)).check(matches(isDisplayed()))
+        onView(withText(R.string.prefs_sync_calendar_contacts_summary)).check(matches(isDisplayed()))
+        assertEquals(PREFERENCE_SYNC_CALENDAR_CONTACTS, prefSync?.key)
+        assertEquals(context.getString(R.string.prefs_sync_calendar_contacts), prefSync?.title)
+        assertEquals(context.getString(R.string.prefs_sync_calendar_contacts_summary), prefSync?.summary)
+        assertTrue(prefSync?.isVisible == true)
+        assertTrue(prefSync?.isEnabled == true)
+
+        onView(withText(R.string.prefs_recommend)).check(matches(isDisplayed()))
+        assertEquals(PREFERENCE_RECOMMEND, prefRecommend?.key)
+        assertEquals(context.getString(R.string.prefs_recommend), prefRecommend?.title)
+        assertEquals(null, prefRecommend?.summary)
+        assertTrue(prefRecommend?.isVisible == true)
+        assertTrue(prefRecommend?.isEnabled == true)
+
+        onView(withText(R.string.prefs_feedback)).check(matches(isDisplayed()))
+        assertEquals(PREFERENCE_FEEDBACK, prefFeedback?.key)
+        assertEquals(context.getString(R.string.prefs_feedback), prefFeedback?.title)
+        assertEquals(null, prefFeedback?.summary)
+        assertTrue(prefFeedback?.isVisible == true)
+        assertTrue(prefFeedback?.isEnabled == true)
+
+        onView(withText(R.string.prefs_privacy_policy)).check(matches(isDisplayed()))
+        assertEquals(PREFERENCE_PRIVACY_POLICY, prefPrivacyPolicy?.key)
+        assertEquals(context.getString(R.string.prefs_privacy_policy), prefPrivacyPolicy?.title)
+        assertEquals(null, prefPrivacyPolicy?.summary)
+        assertTrue(prefPrivacyPolicy?.isVisible == true)
+        assertTrue(prefPrivacyPolicy?.isEnabled == true)
+
+        onView(withText(R.string.prefs_imprint)).check(matches(isDisplayed()))
+        assertEquals(PREFERENCE_IMPRINT, prefImprint?.key)
+        assertEquals(context.getString(R.string.prefs_imprint), prefImprint?.title)
+        assertEquals(null, prefImprint?.summary)
```

Comment:
```suggestion
        assertNull(prefImprint?.summary)
```
Same for others `assertEquals(null, ...)`